rocAL/rocAL/include/pipeline/node.h

```diff
@@ -54,5 +55,6 @@ class Node
     vx_node _node = nullptr;
     size_t _batch_size;
     pMetaDataBatch _meta_data_info;
-    vx_array _src_roi_width, _src_roi_height; // TODO - To be removed after tensor changes in augmentation nodes.
+    vx_scalar _input_layout, _output_layout, _roi_type;
```

Comment:
please check if it is better and possible to have enum instead of scalar for layout, roi_type from this level itself